src/views/DissolutionFirm/DissolutionFirm.vue

```diff
@@ -289,6 +288,10 @@ export default class DissolutionFirm extends Mixins(DateMixin) {
     return this.getValidateSteps && (!this.getCertifyState.certifiedBy || !this.getCertifyState.valid)
   }
 
+  private get dissolutionDate (): string {
+    return this.getDissolutionDate
```

Comment:
You don't need this (or the function on line 357): you can put the Getter/Action right in the component props.